.packit.yaml

```diff
@@ -47,9 +50,14 @@ jobs:
 - <<: *test
   identifier: /hardening/host-os/ansible/stig
   tmt_plan: /hardening/host-os/ansible/stig
+  targets:
+    centos-stream-9: {}
```

Comment:
Let's try running it also on cs8 to see how long it takes when it's in a single job. Previously, the ansible stig was excluded from CS8 because it took too much, but I'd like to give it another try.

Reply:
Shows outdated, but it's still there. No need to try it in this PR, it's just a note for future to give STIG one another shot on CS8 in CI.